common-graphql/src/main/scala/queries/schema/UserPreferencesDB.scala

```diff
@@ -24,4 +26,10 @@ trait UserPreferencesDB:
   object Enums:
     type ItcChartType            = enums.ItcChartType
     type LucumaTableIdsEnum      = explore.model.enums.TableId
-    type LucumaSortDirectionEnum = explore.model.enums.SortDirection
+    type LucumaSortDirectionEnum = SortDirection
+
+  given Enumerated[SortDirection] =
```

Comment:
Why not to define the enumerated where `SortDirection` is defined?

Reply:
`SortDirection` is defined in the facade now, which doesn't have `lucuma-core` as dependency.